packages/mail-client/src/mail-client.ts

```diff
@@ -232,21 +285,25 @@ function teardown(transport: Transporter<SentMessageInfo>, socket?: Socket) {
  * Builds a transport between the application and the mail server, sends mails sequentially by using the transport, then closes it.
  * This function also does the destination look up, when passing `DestinationOrFetchOptions`.
  * @param destination - A destination or a destination name and a JWT.
+ * @param mailClientOptions - A {@link MailClientOptions} that defines the configurations of the mail client, e.g., whether the mails are sent in parallel.
  * @param mailOptions - Any objects representing {@link MailOptions}. Both array and varargs are supported.
  * @returns A promise resolving to an array of {@link MailResponse}.
  * @see https://sap.github.io/cloud-sdk/docs/js/features/connectivity/destination#referencing-destinations-by-name
  */
 export async function sendMail<T extends MailOptions>(
   destination: DestinationOrFetchOptions,
+  mailClientOptions: MailClientOptions,
   ...mailOptions: T[]
 ): Promise<MailResponse[]>;
 export async function sendMail<T extends MailOptions>(
   destination: DestinationOrFetchOptions,
+  mailClientOptions: MailClientOptions,
   mailOptions: T[]
 ): Promise<MailResponse[]>;
 /* eslint-disable jsdoc/require-jsdoc */
 export async function sendMail<T extends MailOptions>(
   destination: DestinationOrFetchOptions,
+  mailClientOptions: MailClientOptions,
```

Comment:
[pp] I would prefer the order sendMail(destination:Destiantion,mails:MailOptions[],mailClientOptions?:Options)). Doing so you can skip the options for the client which are options. You have then to pass an array for the mails but this would be fine I think.

Reply:
If more and more people are using typescript the people see directly in the compile step if it is an array or a single object.